content/accessibility/semantic-html.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Semantic HTML
+---
+
+## Overview
+
+Semantic HTML provides meaning to the content of a web page. It involves using the correct HTML element for the job. Semantic HTML breaks the page up into meaningful sections.
+
+## Why?
+
+Many groups of people benefit from properly used semantic HTML. Using the correct elements allows assistive technology to accurately convey the purpose of the content to the user. Without it, they will not be able to navigate easily. Other benefits of using semantic HTML are SEO and code readability.
+
+## How to test for semantic HTML
+
+HTML validators, automated tooling, unit tests, and linters are a few ways to test for proper HTML. Most automated tools can catch some incorrect usage of HTML elements. For example, if heading levels are out of order, it will be reported as a violation. Keep in mind that [automated tools only catch around 20-25% of accessibility errors](https://www.essentialaccessibility.com/blog/automated-accessibility-testing-tools-how-much-do-scans-catch#:~:text=For%20WCAG%202.1%20AA%20tests,by%20yes%20or%20no%20conditions.).
+
+This is a subset of the tools available:
+
+- [HTML validator](https://validator.w3.org/)
+- Automated tools
+    - [aXe DevTools](https://www.deque.com/axe/devtools/)
+    - [Lighthouse](https://github.com/GoogleChrome/lighthouse)
+- Linting
+    - [Primer ViewComponents](https://github.com/primer/view_components/blob/main/docs/content/linting.md)
+    - [Primer React](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md#linting)
+    - [eslint-plugin-jsx-a11y](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y)
+
+## Guidelines
+
+### For designers
+
+In your designs, annotate what HTML elements should be used for various parts of the design, if appropriate. Understand that most designs are achievable with CSS alone, agnostic of the HTML elements used. If a component has state, annotate the ARIA attributes needed for each state. For example, for an accordion, indicate that the header should have `aria-expanded="true"` when the panel is expanded, and `aria-expanded="false"` when collapsed.
```

Comment:
Can true semantics indicate state? I'm waffling here on whether to dive into `ARIA` examples on this Semantics overview.

Reply:
Thanks for bring this up! Now re-reading it with your feedback, I think I should take that part out. I feel it's not really semantic related, but more of a best practice when using ARIA.

Reply:
Ok 👍